### ISSUE TITLE
Missing version in custom .htaccess builds 

### ISSUE THREAD

**Issue by florianbouvot** [h5bp/server-configs-apache#308]
Hi, we are using custom build in our tools and I try to update to follow last update and version doesn't appear:

<img width="680" alt="Capture d’écran 2022-05-11 à 10 28 32" src="https://user-images.githubusercontent.com/2061540/167804665-d5ae1142-3330-4065-8f2c-fde75ed4b583.png">

**Comment by LeoColomb** [comment]
Thanks for raising this issue @florianbouvot.
The version is extracted from the `package.json` file located at the root of the repository.

https://github.com/h5bp/server-configs-apache/blob/165314100d24dbb257e1707541da16ded01de7cb/bin/build.sh#L17-L18

Do you confirm the file does exist on your side?

**Comment by florianbouvot** [comment]
@LeoColomb the output

![Capture d’écran 2022-05-11 à 18 59 49](https://user-images.githubusercontent.com/2061540/167906084-8a924d83-7fee-4f79-8e64-e424eb3fd085.png)

**Comment by LeoColomb** [comment]
Yes, here is the reason why: `realpath` command is not found.
Introduced by https://github.com/h5bp/server-configs-apache/commit/93d524f37b4a7b393ebfbf50aa5fb13d0967cbee with https://github.com/h5bp/server-configs-apache/pull/295
@tomkyle Any idea?

**Comment by tomkyle** [comment]
OK, so the command is missing on that system. Seems to me though that the realpath command is nothing new in our *nix eco systems as it stems from 2007 or 2008 [1] and also found its way into GNU coreutils 10 years ago [2]. 

As a quick fix for @florianbouvot I'd propose installing the realpath command (simple one-liner) [3], so he's up and running again in 5 minutes.

Because of the ages-old availability of this command I do not feel a sophisticated fallback solution is necessary. But a nicer error handling could improve screen experience, perhaps something like this:

```bash
function require  {
	command -v $1 >/dev/null 2>&1 || {
		echo >&2 "Command '${1}' is required but not installed."
		exit 1;
	}
}

require realpath
```

If needed I can provide PR this weekend.

[1] https://stackoverflow.com/a/284664
[2] https://github.com/coreutils/coreutils/commit/77ea441f79aa115f79b47d9c1fc9c0004c5c7111
[3] https://command-not-found.com/realpath

**Comment by nosilleg** [comment]
@florianbouvot You are using MacOs which doesn't have `realpath` by default. If you have Homebrew installed then `brew install coreutils` will solve your issue.

If you don't have Homebrew installed then I would recommend installing Homebrew. 😃

**Comment by florianbouvot** [comment]
@nosilleg yes I'm using macOS and of course Home-brew, it's the first time that it doesn't works.
I will try with `coreutils`.
